strings.asm

```diff
@@ -80,10 +82,11 @@ s_wid_root:   .text "Root ", 0       ; Wordlist ID 3, note space at end
 s_see_nt:     .text "nt: ", 0
 s_see_xt:     .text "xt: ", 0
 s_see_size:   .text "size (decimal): ", 0
+s_see_cfapfa: .text "as CFA: 3 PFA: ", 0
```

Comment:
This is for #120, which shows CFA/PFA breakdown after size for HC words.
```
42 constant life  ok
see life 
nt: 800  xt: 80C 
flags: CO 0 IM 0 AN 0 NN 1 | UF 0 HC 1 ST 0 
size (decimal): 5 as CFA: 3 PFA: 2 

080C  20 61 D7 2A 00   a.*.

80C   D761 jsr     
``` 
I couldn't find a simple way to have CFA shown after the jsr without adding a bunch of special case code to disasm.

Reply:
I think that's fine.